es6-arrow-functions/main.js

```diff
@@ -0,0 +1,45 @@
+const numbers = [12, 11, 10, 9, 8, 7, 6];
+const book = {
+  name: 'Bea Wolf',
+  author: 'Zach Weinersmith',
+  illustrator: 'Boulet',
+  published: 2023,
+};
+
+console.log('Exercise 1');
+const doubled = numbers.map((n) => {
+  return n * 2;
+});
+console.log('doubled', doubled);
+
+console.log('\n\nExercise 2');
+function actOnEntries(obj, action) {
+  Object.entries(obj).forEach((e) => {
+    action(e);
+  });
+}
+actOnEntries(book, (entry) => {
+  console.log(`${entry[0]} = ${entry[1]}`);
+});
+
+console.log('\n\nExercise 3');
+const foo = () => {
+  console.log("Why doesn't this log???");
+};
+
+foo();
```

Comment:
In the original code, code was never called. It was only referenced in `line 31` below!